zhaquirks/tuya/tuya_motion.py

```diff
@@ -1436,6 +1436,7 @@ class TuyaMotionDetectionMode(t.enum8):
 (
     TuyaQuirkBuilder("_TZE200_2aaelwxk", "TS0601")
     .applies_to("_TZE200_kb5noeto", "TS0601")
+    .applies_to("HOBEIAN", "ZG-204ZM")
```

Comment:
I think this change may be accidentally included in this PR

Reply:
Accidental commits were reverted.